modules/kafka/benchmark_test.go

```diff
@@ -0,0 +1,37 @@
+package kafka_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/kafka"
+)
+
+func startStopBenchmark(b *testing.B, image string) {
+	b.Helper()
+	for b.Loop() {
+		kafkaContainer, err := kafka.Run(context.Background(),
+			image,
+		)
+		if err != nil {
+			b.Fatalf("failed to start container: %s", err)
+		}
```

Comment:
suggestion: use testify require to simplify, more below

```suggestion
		require.NoError(b, err)
```